### ISSUE TITLE
didnot support WebGL 1

### ISSUE THREAD

**Issue by NateLing** [servo/pathfinder#433]
![image](https://user-images.githubusercontent.com/9606668/89850515-1a041b00-dbbd-11ea-9904-e8e5608aac21.png)

**Comment by s3bk** [comment]
Yes, WebGl1 is not supported. Pathfinder needs OpenGL (ES) 3.0 at minimum.
